### PR TITLE
Add Memgraph Lab v1.1.0

### PR DESCRIPTION
**Pull request by gitbuda**

### CODE REVIEW

File: products/lab/v1.1.0/how_to_install_memgraph_lab_on_macos.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+Download Memgraph Lab by visiting [the download
+page](https://memgraph.com/download/#tools).
+
+![](data/download_lab.png)
+
+Once you have Memgraph Lab installed, run the app and you should be presented
+with a connect screen. The username and password fields are only for Memgraph
+Enterprise users and are not required if you’re using Memgraph Core.
+
+The default connection string set to "bolt://localhost:7687". If you're using a
```
Comment (ipaljak) [inline products/lab/v1.1.0/how_to_install_memgraph_lab_on_macos.md]: I would style the connection string as `bolt://localhost:7687`.

File: products/lab/v1.1.0/how_to_install_memgraph_lab_on_macos.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+different port you will have to change the connection string to point to that
+port. i.e. "bolt://localhost:<your port number>"
```
Comment (ipaljak) [inline products/lab/v1.1.0/how_to_install_memgraph_lab_on_macos.md]: Same as above.

File: products/lab/v1.1.0/how_to_install_memgraph_lab_on_macos.md (hunk 1 above)
Comment (gitbuda) [reply products/lab/v1.1.0/how_to_install_memgraph_lab_on_macos.md]: Done.

File: products/lab/v1.1.0/how_to_install_memgraph_lab_on_macos.md (hunk 2 above)
Comment (gitbuda) [reply products/lab/v1.1.0/how_to_install_memgraph_lab_on_macos.md]: Done.

**Pull request merged**